static/js/settings_config.js

```diff
@@ -193,6 +193,22 @@ const time_limit_dropdown_values = new Map([
         },
     ],
 ]);
+
+const oss_showcase_values = new Map([
+    [
+        "hidden",
+        {
+            text: i18n.t("No"),
+        },
+    ],
+    [
+        "visible",
+        {
+            text: i18n.t("Yes"),
+        },
+    ],
+]);
```

Comment:
If you follow my previous comment then you have to change this value map too, make it something like `email_address_visibility_values`